src/applications/mhv-landing-page/components/LandingPageAlerts.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IdentityNotVerified from '~/platform/user/authorization/components/IdentityNotVerified';
+import CardLayout from './CardLayout';
+import MhvRegistrationAlert from './MhvRegistrationAlert';
+import UnregisteredAlert from './UnregisteredAlert';
+import MhvBasicAccountAlert from './MhvBasicAccountAlert';
+import VerifyAndRegisterAlert from './VerifyAndRegisterAlert';
+
+const LandingPageAlerts = ({
+  registered,
+  verified,
+  userHasMhvAccount,
+  userHasMhvBasicAccount,
+  showsVerifyAndRegisterAlert,
+  signInService,
+  cards,
+  unVerifiedHeadline,
+}) => {
+  const noCardsDisplay = verified ? (
+    <UnregisteredAlert />
+  ) : (
+    <IdentityNotVerified
+      headline={unVerifiedHeadline}
+      showHelpContent={false}
+      showVerifyIdenityHelpInfo
+      signInService={signInService}
+    />
+  );
+
+  return (
+    <>
+      {registered && !userHasMhvAccount && <MhvRegistrationAlert />}
+      {registered ? <CardLayout data={cards} /> : noCardsDisplay}
```

Comment:
I'm not sure moving the cards inside the `LandingPageAlerts` component makes intuitive sense. Could we keep the cards outside of this component and in here do something like:

```suggestion
      {!registered && noCardsDisplay}
```